corehq/ex-submodules/couchforms/util.py

```diff
@@ -465,6 +465,11 @@ def process(xform):
                             instance._id = new_id
                             instance.problem = error_message
                             instance.save()
+                            if instance.orig_id == "8410bd60-2269-4c5d-b262-5cf828cfddee":
```

Comment:
should we leave a comment in here? maybe something like "if it is past feb 20 2015 and this line of code is still in the repo you should delete it" :sunglasses: